### ISSUE TITLE
Doesn't seem to work properly with the newer versions of gamescope

### ISSUE THREAD

**Issue by zastrixarundell** [flathub/org.freedesktop.Platform.VulkanLayer.MangoHud#58]
This is basically the MangoHud repo's version of: https://github.com/flathub/org.freedesktop.Platform.VulkanLayer.gamescope/issues/264.

What happens is that when I try to run gamescope with `--mangoapp`, the configuration is not read properly. Initially I thought that it was an issue with gamescope because when I downgrade it to a specific version it will read the MangoHud config file.

But I also saw that even with the non-broken version, when I hide the UI and try to re-enable it, it won't pop back in. 

In the tests I'll use Elden Ring while having `MANGOHUD=1` env override set in steam.

When I try to use gamescope via `gamescope -h 1440 -H 1080 -f -r 60 -- %command%` and basic MangoHud inside the game (not gamescope), I get normal, expected, behavior where I can enable/disable it however I please and it reads my config file:
![image](https://github.com/user-attachments/assets/9cc6dd44-62e3-457c-be0d-358b14ebccf4)

When I try to use gamescope with `--mangoapp`, so: `gamescope -h 1440 -H 1080 -f -r 60 --mangoapp -- %command%`, I will get the following behavior:
![image](https://github.com/user-attachments/assets/20a23ac8-a0e8-4b13-9eb7-23bd55edafe1)

> Having the 2 gamescope windows is fine afaik, because I have `MANGOHUD=1` setup

As you can see, the config file is not loaded. When I try to press `RShift+F12`x2 (to hide and show the menu back up) I get this:
![image](https://github.com/user-attachments/assets/fc31789c-eb05-4b49-bb8f-9a197e033a5a)

As it can be seen, the gamescope's MangoHud window is now missing.

**Comment by jntesteves** [comment]
Seems fixed for me on version 0.8.0. I've just merged it on branch 24.08, Flathub with likely build and publish it tonight. Can you please test again when you get the update?

**Comment by zastrixarundell** [comment]
I gladly would but seems like flathub's servers errored out during the publish step (error 500 on their publish API endpoint). The CD probably needs to be restarted.

**Comment by jntesteves** [comment]
Yeah, Flathub infra is currently broken for publishing extensions, and they didn't give an estimate of when they'll be able to work on it. If you want to test the latest version right now, you can install the build in the PR [60], otherwise we'll have to wait.

[60]: https://github.com/flathub/org.freedesktop.Platform.VulkanLayer.MangoHud/pull/60#event-16268603268

**Comment by zastrixarundell** [comment]
Good news! Installed it via the linked PR and everything seems fine. For some reason I can't upload a video but it showed the info:
![Image](https://github.com/user-attachments/assets/0cf1199b-7ea1-439d-ad8a-9dc61a2a20a8)

I was able to close/reopen mangohud as well.

I won't close the issue before I test it out on the main distributed binary, but it seems it'll most likely work.

**Comment by zastrixarundell** [comment]
Yep. MangoHud is now updated to 0.8.0 and everything seems to be in line! Thank you very much!
